docs/source/yelpsoa_configs.rst

```diff
@@ -127,6 +133,78 @@ Marathon, Tron, or ``paasta remote-run``.
     a host is both whitelisted and blacklisted, the blacklist will take precedence.  Only one location type
     of whitelisting may be specified.
 
+  * ``pool``: The pool of machines a PaaSTA app runs in. If no pool is set,
+    an app will automatically be set to run in ``default`` pool.
+
+    Warning: In order for an service to be launched in a particular pool, there
+    *must* exist some nodes that already exist with that particular
+    pool attribute set.
+
+Kubernetes
+^^^^^^^^^^
+
+These options are only applicable to tasks scheduled on Kubernetes.
+
+  * ``node_selectors``: A map of labels a node is required to have for a task
+    to be launched on said node. There are several ways to define a selector.
+    The simplest is a key-value pair. For example, this selector restricts a
+    task to c3.8xlarge instances::
+
+      node_selectors:
+        instance_type: c3.8xlarge
+
+    The value can also be a list of multiple values. For example, this selector
+    restricts a task to both c3.8xlarge and m5.2xlarge instances::
+
+      node_selectors:
+        instance_type: ["c3.8xlarge", "m5.2xlarge"]
+
+    For more complex cases, an operator, and optionally a value (or values) must
+    be set. Here are some examples:
+
+    * Restricts a task to instances that are not c3.8xlarges::
+
+        node_selectors:
+          instance_type:
+            - operator: NotIn
+              values: ["c3.8xlarge"]
+
+    * Requires that a node have the ``vip`` label::
+
+        node_selectors:
+          vip:
+            - operator: Exists
+
+    * Requires that a node not have the ``vip`` label::
+
+        node_selectors:
+          vip:
+            - operator: DoesNotExist
+
+    * Requires that a node have a priority greater than 1 and less than 5::
+
+        node_selectors:
+          priority:
+            - operator: Gt
+              value: 1
+            - operator: Lt
+              value: 5
```

Comment:
I am not sure I understand this part. How do we assign the priority to nodes? And what's the scenario you expect users would need this?

Reply:
I'm just using priority as an example of a label that could have a numeric value. I don't envision any particular scenario where we will end up using the Gt or Lt operators, but they do exist in k8s, so I just added them. We can remove them if we don't want to support them.

Reply:
I am not sue if this would confuse users, but I think we can remove it anytime if we want.